maintenance_team_hierarchy/models/maintenance_team.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MaintenanceTeam(models.Model):
+
+    _parent_name = "parent_id"
+    _parent_store = True
+    _parent_order = "name"
+    _inherit = "maintenance.team"
+
+    parent_id = fields.Many2one("maintenance.team", ondelete="restrict")
+    parent_path = fields.Char(index=True)
+
+    request_ids = fields.Many2many(
+        "maintenance.request", compute="_compute_request_ids",
+    )
+
+    todo_request_ids = fields.One2many(compute="_compute_new_todo_requests")
+    todo_request_count = fields.Integer(compute="_compute_new_todo_requests")
+    todo_request_count_date = fields.Integer(compute="_compute_new_todo_requests")
+    todo_request_count_high_priority = fields.Integer(
+        compute="_compute_new_todo_requests"
+    )
+    todo_request_count_block = fields.Integer(compute="_compute_new_todo_requests")
+    todo_request_count_unscheduled = fields.Integer(
+        compute="_compute_new_todo_requests"
+    )
```

Comment:
Is it possible to show on views these fields? If they are not shown, I don't know what their purpose is.

Reply:
These fields aren't new, we have just changed its compute function. They are already shown in the kanban view.

Reply:
Reviewed on runbot, I don't see them in the kanvan view. Actually, there's no kanban view for `maintenance.team` in this addon

Reply:
They are in the odoo core `maintenance` module. https://github.com/odoo/odoo/blob/13.0/addons/maintenance/views/maintenance_views.xml#L807

Reply:
You're right, I was looking at kanban view of configuration menu, not dashboard one, sorry.